x-pack/test/apm_api_integration/tests/service_groups/save_service_group.spec.ts

```diff
@@ -65,12 +65,16 @@ export default function ApiTest({ getService }: FtrProviderContext) {
     afterEach(deleteServiceGroups);
 
     it('creates a new service group', async () => {
-      const response = await callApi({
+      const serviceGroup = {
         groupName: 'synthbeans',
         kuery: 'service.name: synth*',
-      });
+      };
+      const response = await callApi(serviceGroup);
       expect(response.status).to.be(200);
-      expect(Object.keys(response.body).length).to.be(0);
+      expect(response.body).to.have.property('id');
+      expect(response.body).to.have.property('groupName', serviceGroup.groupName);
+      expect(response.body).to.have.property('kuery', serviceGroup.kuery);
+      expect(response.body).to.have.property('updatedAt');
```

Comment:
Shouldn't this test also retrieve the service group to make sure it was created?